### DISCUSSION TITLE
Configuration for DA namespace migrations

### DISCUSSION THREAD

**Discussion by mzabaluev** [movementlabsxyz/movement#473] [Ideas]
Parent discussion: #360
Tracking issue: #474

To evolve our blob format on Celestia DA, it's been suggested to switch namespaces when migrating between backward-incompatible blob formats.
Before governance is implemented, the nodes can use a configuration file (or, more generally, a serde schema) to map migration heights to Celestia namespace IDs. The well-known configuration file is shared between the Suzuka node and the DA light node. 

The format can be as simple as (in JSON):

```json
{
  "version": 1,
  "migrations": {
    "100342": "deadbeef0123456789abcdef",
    "117683": "abcdef9876543210beefdead",
  }
}
```

**Comment by l-monninger** [comment]
Consider whether #470 can be moved over here.

**Reply by mzabaluev** [reply]
I think it's better to close this in favor of #470.
